api-metastore/src/test/java/org/zalando/nakadi/controller/PartitionsControllerTest.java

```diff
@@ -195,28 +195,6 @@ public void whenGetPartitionThenOk() throws Exception {
                 .andExpect(content().string(TestUtils.JSON_TEST_HELPER.matchesObject(TEST_TOPIC_PARTITION_0)));
     }
 
-    @Test
-    public void whenUnauthorizedGetPartitionThenForbiddenStatusCode() throws Exception {
-        Mockito.when(eventTypeCacheMock.getEventType(TEST_EVENT_TYPE)).thenReturn(EVENT_TYPE);
-
-        Mockito.doThrow(TestUtils.mockAccessDeniedException()).when(authorizationValidator).authorizeStreamRead(any());
-
-        mockMvc.perform(
-                get(String.format("/event-types/%s/partitions/%s", TEST_EVENT_TYPE, TEST_PARTITION)))
-                .andExpect(status().isForbidden());
-    }
-
-    @Test
-    public void whenUnauthorizedGetPartitionsThenForbiddenStatusCode() throws Exception {
-        Mockito.when(eventTypeCacheMock.getEventType(TEST_EVENT_TYPE)).thenReturn(EVENT_TYPE);
-
-        Mockito.doThrow(TestUtils.mockAccessDeniedException()).when(authorizationValidator).authorizeStreamRead(any());
-
-        mockMvc.perform(
-                get(String.format("/event-types/%s/partitions", TEST_EVENT_TYPE)))
-                .andExpect(status().isForbidden());
-    }
-
```

Comment:
Is the response from `VIEW` checked somewhere in the test?

Reply:
I added a error test case for View in PartitionsController, this one checks for the response.